src/a11y-meta.yml

```diff
@@ -637,6 +718,9 @@
 "pulse":
     title: "Pulse Icon"
     description: "A picture depicting a cardiogram."
+"radiator":
+    title: "Radiator Icon"
+    description: "A picture showing a radiator in a house."
```

Comment:
"A picture showing a radiator used for heating."

Reply:
done